Directory.Build.props

```diff
@@ -77,6 +77,11 @@
     <LicenseUrl>https://github.com/dotnet/runtime/blob/master/LICENSE.TXT</LicenseUrl>
     <RuntimeIdGraphDefinitionFile>$([MSBuild]::NormalizePath('$(LibrariesProjectRoot)', 'pkg', 'Microsoft.NETCore.Platforms', 'runtime.json'))</RuntimeIdGraphDefinitionFile>
     <LicenseFile>$(RepoRoot)LICENSE.TXT</LicenseFile>
+    <PackageLicenseFile>$(RepoRoot)LICENSE.TXT</PackageLicenseFile>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <Copyright>$(CopyrightNetFoundation)</Copyright>
```

Comment:
cant we keep these properties in packaging.props  as these are specific to packaging ? and just import that file everywhere ?

Reply:
eng/packaging.props and eng/packaging.targets contain libraries specifics that don't make sense to be imported everywhere. We already define some of the packaging properties in Directory.Build.props/targets so I don't see a problem with that.